server/commands/teamCreator.ts

```diff
@@ -33,7 +35,9 @@ async function teamCreator({
   ip,
 }: Props): Promise<TeamCreatorResult> {
   let authP = await AuthenticationProvider.findOne({
-    where: authenticationProvider,
+    where: id
+      ? { ...authenticationProvider, teamId: id }
+      : authenticationProvider,
```

Comment:
🥦 2/ do a best effort here to find the right team. If a `teamId` is provided, we're interpreting that as the only team we're interested in.